packages/ui/src/StandardUIViewer.js

```diff
@@ -222,12 +225,14 @@ class StandardUIViewer extends mapray.RenderCallback
         var self = this;
 
         window.addEventListener( "blur", function ( event ) { self._onBlur( event ); }, { passive: false } );
-        canvas.addEventListener( "mousedown", function ( event ) { self._onMouseDown( event ); }, { passive: false } );
-        document.addEventListener( "mousemove", function ( event ) { self._onMouseMove( event ); }, { passive: false } );
-        document.addEventListener( "mouseup", function ( event ) { self._onMouseUp( event ); }, { passive: false } );
-        document.addEventListener( "wheel", function ( event ) { self._onMouseWheel( event ); }, { passive : false } );
-        document.addEventListener( "keydown", function ( event ) { self._onKeyDown( event ); }, { passive: false } );
-        document.addEventListener( "keyup", function ( event ) { self._onKeyUp( event ); }, { passive: false } );
+        canvas.addEventListener( "mousedown", function ( event ) { self._onMouseDown( event ); }, { passive: true } );
+        canvas.addEventListener( "mousemove", function ( event ) { self._onMouseMove( event ); }, { passive: true } );
+        document.addEventListener( "mousemove", function ( event ) { self._onMouseMove( event ); }, { capture: true } );
+        canvas.addEventListener( "mouseup", function ( event ) { self._onMouseUp( event ); }, { passive: true } );
+        document.addEventListener( "mouseup", function ( event ) { self._onMouseUp( event ); }, { capture: false } );
+        canvas.addEventListener( "wheel", function ( event ) { self._onMouseWheel( event ); }, { passive : false } );
+        canvas.addEventListener( "keydown", function ( event ) { self._onKeyDown( event ); }, { capture: false, passive: false } );
+        canvas.addEventListener( "keyup", function ( event ) { self._onKeyUp( event ); }, { passive: true } );
     }
```

Comment:
Bind window-level event listeners for move and up/end events. the pointer capture API is not supported by all necessary platforms, window-level event listeners is the best way to fall outside the map canvas element.
Using capture:true for the move event prevent map move events from being fired during a drag.